android/app/src/main/java/org/fossasia/openevent/viewmodels/auth/LoginActivityViewModel.java

```diff
@@ -0,0 +1,63 @@
+package org.fossasia.openevent.viewmodels.auth;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.auth.Login;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
+
+import static org.fossasia.openevent.utils.AuthUtil.INVALID;
+import static org.fossasia.openevent.utils.AuthUtil.VALID;
+
+public class LoginActivityViewModel extends ViewModel {
+
+    private MutableLiveData<LoginResponsePair> loginResponse;
+    private final CompositeDisposable compositeDisposable = new CompositeDisposable();
+
+    public LiveData<LoginResponsePair> loginUser(String email, String password) {
+        if (loginResponse == null) {
+            loginResponse = new MutableLiveData<>();
+        }
+        compositeDisposable.add(APIClient.getOpenEventAPI().login(new Login(email, password))
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(response -> {
+                            Timber.d("Saved token and logged in successfully");
+                            loginResponse.setValue(new LoginResponsePair(VALID, response.getAccessToken()));
+                        }, throwable -> {
+                            Timber.e(throwable.toString());
+                            loginResponse.setValue(new LoginResponsePair(INVALID, ""));
+                        }));
+        return loginResponse;
+    }
+
+    @Override
+    protected void onCleared() {
+        compositeDisposable.dispose();
+        super.onCleared();
+    }
+
+    public class LoginResponsePair{
```

Comment:
Remove the word pair, make class static use lombok, make fields final, and space after r